### DISCUSSION TITLE
Error... Exception Object ( [message:protected] => [string:Exception:private] => [code:protected] => 502 [file:protected] => /

### DISCUSSION THREAD

**Discussion by yefridho** [kirimwa-id/forum#190] [Tanya Jawab]
Mas Rio

Ini kenapa ya mas....
Tiba2 error begini semua....

Pernah terjadi beberapa kali. kemudian bener sendiri.
Apakah ini masalah server atau bagaimana mas.
Mohon bantuannya segera ya mas Rio, karena klien kami membutuhkan WA aktif lagi segera...

Terima kasih banyak Mas

**Comment by yefridho** [comment]
Mas Rio...

Udah seminggu gak ada bantuan sedikitpun ini mas ?

ini tiba-tiba error [Exception Object ( [message:protected] => [string:Exception:private] => [code:protected] => 502 [file:protected] => /](https://github.com/kirimwa-id/forum/discussions/190#top)

kenapa ini mas...
mohon jawabannya mas...

**Comment by pst1812** [comment]
saya juga punya problem yang sama, gimana ya mas rio? mohon bantuannya

**Comment by yefridho** [comment]
Sepertinya ini karena masalah error server.... tiba2 error.... dan tiba2 bisa...

**Reply by yefridho** [reply]
mas error begini lagi mas. 

Exception Object ( [message:protected] => error code: 500 [string:Exception:private] => [code:protected] => 500 [file:protected] => /var/www/html/contoh/contoh_wa.php [line:protected] => 49 [trace:Exception:private] => Array ( [0] => Array ( [file] => /var/www/html/contoh/contoh_wa.php [line] => 271 [function] => apiKirimWaRequest [args] => Array ( [0] => Array ,,,,

**Comment by yefridho** [comment]
mas minta tolong mas. error begini lagi.

Exception Object ( [message:protected] => error code: 500 [string:Exception:private] => [code:protected] => 500 [file:protected] => /var/www/html/contoh/contoh_wa.php [line:protected] => 49 [trace:Exception:private] => Array ( [0] => Array ( [file] => /var/www/html/contoh/contoh_wa.php [line] => 271 [function] => apiKirimWaRequest [args] => Array ( [0] => Array ,,,,

masalahnya dimana ya mas ?

**Comment by yefridho** [comment]
Mas Rio

Mohon Bantuannya...
Error ini terjadi lagi mas....
Mohon solusinya mas...

Exception Object ( [message:protected] => {"message":"Failed to send message. Timeout, can not fetch image URL."} [string:Exception:private] => [code:protected] => 500 [file:protected] => /var/www/html/contoh/contoh_wa.php [line:protected] => 49 [trace:Exception:private] => Array ( [0] => Array ( [file] => /var/www/html/contoh/contoh_wa.php [line] => 208 [function] => apiKirimWaRequest [args] => Array (.........

Moho solusinya ya mas....
